README.md

```diff
@@ -35,12 +35,41 @@ http_archive(
 )
 
 load("@io_bazel_rules_play_routes//:workspace.bzl", "play_routes_repositories")
-play_routes_repositories()
+play_routes_repositories("2.7")
+load("@play_routes//:defs.bzl", play_routes_pinned_maven_install = "pinned_maven_install")
+play_routes_pinned_maven_install()
+
+bind(
+  name = "default-play-routes-compiler-cli",
+  actual = "@io_bazel_rules_play_routes//default-compiler-clis:scala_2_12_play_2_7"
+)
+
+```
+
+This installs `rules_play_routes` to your `WORKSPACE` and binds the default play routes compiler cli the rules will use. Update the commit as needed.
+
+In the above example, the play routes compiler cli for Scala 2.12 and Play 2.7 is used. However, you can specify a different compiler.
+
+We provide 5 default compilers:
+
+- For Scala 2.11 + Play 2.5: `@io_bazel_rules_play_routes//default-compiler-clis:scala_2_11_play_2_5`
+- For Scala 2.11 + Play 2.6: `@io_bazel_rules_play_routes//default-compiler-clis:scala_2_11_play_2_6`
+- For Scala 2.11 + Play 2.7: `@io_bazel_rules_play_routes//default-compiler-clis:scala_2_11_play_2_7`
+- For Scala 2.12 + Play 2.6: `@io_bazel_rules_play_routes//default-compiler-clis:scala_2_12_play_2_6`
+- For Scala 2.12 + Play 2.7: `@io_bazel_rules_play_routes//default-compiler-clis:scala_2_12_play_2_7`
+
+To bind one of the default compilers, simply specify the correct Play version in the call to `play_routes_repositories` and update the bind statement:
```

Comment:
Would it be better to make it clear Scala 2.11 and 2.12 will be provided once Play 2.6 or 2.7 is specified? Basically let user know why there is no config for Scala version.